.mvn/maven-build-cache-config.xml

```diff
@@ -82,6 +75,9 @@
                     <reconciles>
                         <reconcile propertyName="skip" skipValue="true"/>
                     </reconciles>
```

Comment:
I (Informational): This ignores the deprecated `commandLineRules` property, triggering warnings to display while executing the `install` phase of the build.

```
[WARNING] Cannot find a Mojo parameter 'commandLineRules' to read for Mojo org.apache.maven.plugins:maven-enforcer-plugin:3.5.0:enforce {execution: enforce-maven}. This parameter should be ignored.
[WARNING] Cannot find a Mojo parameter 'commandLineRules' to read for Mojo org.apache.maven.plugins:maven-enforcer-plugin:3.5.0:enforce {execution: enforce-java}. This parameter should be ignored.
```